src/explore-education-statistics-common/src/components/form/FormSelect.tsx

```diff
@@ -57,69 +60,104 @@ const FormSelect = ({
 }: FormSelectProps) => {
   return (
     <>
-      <FormLabel id={id} label={label} hideLabel={hideLabel} />
+      <FormSelectWrapper inline={inline}>
```

Comment:
Don't need to split out a separate component for this. It's a bit of unnecessary indirection.

Would just implement this as a wrapping `div` with conditional `className` for simplicity.

Reply:
Unnecessary divs make me sad.